src/azul/project/hca/metadata_generator.py

```diff
@@ -0,0 +1,197 @@
+# Adapted from https://github.com/HumanCellAtlas/\
```

Comment:
It would probably be more useful to not wrap this line despite it being very long. Makes it easier to copy and paste it.